dashboard/app/views/levels/editors/_ailab.html.haml

```diff
@@ -0,0 +1,10 @@
+= render partial: 'levels/editors/fields/instructions_area', locals: {f: f}
+= render partial: 'levels/editors/fields/teacher_only_markdown', locals: {f: f}
+= render partial: 'levels/editors/fields/code_area', locals: {f: f}
+= render partial: 'levels/editors/fields/video', locals: {f: f}
+= render partial: 'levels/editors/fields/bubble_choice_sublevel', locals: {f: f}
+= render partial: 'levels/editors/fields/special_level_types', locals: {f: f}
```

Comment:
What from the special level types are you using here?

Reply:
Oh I was just borrowing these from Web Lab as well.  I can remove.